NotebookAnalyzers/FunctionsNumberExtraction.ipynb

```diff
@@ -0,0 +1,82 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "statutory-helicopter",
+   "metadata": {},
+   "source": [
+    "# Number of functions extraction"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "id": "institutional-castle",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import utils"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "id": "norwegian-finance",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "code = utils.notebookToCode(\"my-attempt-at-analytics-vidhya-job-a-thon.ipynb\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "id": "sorted-shock",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "3"
+      ]
+     },
+     "execution_count": 25,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "functionsNumber(code)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "intimate-lambda",
+   "metadata": {},
+   "outputs": [],
+   "source": []
```

Comment:
Mantieni il notebook pulito quando lo condividi con altri: le celle vuote o non eseguite alla fine vanno eliminate. È una delle best practice note in letteratura che sicuramente citerai nella tesi :)